### PR TITLE
doc: extend Thread CLI sample with SRP information

### PR DESCRIPTION
**Pull request by edmont**
Show the basic usage of SRP commands.

Fix some output changes from existing commands.

Signed-off-by: Eduardo Montoya <eduardo.montoya@nordicsemi.no>

KRKNWK-13754

### CODE REVIEW

File: samples/openthread/cli/README.rst [hunk 1]
```diff
@@ -454,14 +459,65 @@ To test the Thread 1.2 features, complete the following steps:
...
          16 bytes from fe80:0:0:0:acbd:53bf:1461:a861: icmp_seq=3 hlim=64 time=421ms
          1 packets transmitted, 1 packets received. Packet loss = 0.0%. Round-trip min/a
          Done
 
       Observe that the reply latency is reduced to a value below 500 ms.
       The reduction occurs because the transmission from the leader is performed using CSL, based on the CSL Information Elements sent by the CSL Receiver.
 
+#. Verify the Service Registration Protocol (SRP) functionality.
+
+   a. On the leader kit, enable the Backbone Router function:
```
Comment (LuDuda) [inline samples/openthread/cli/README.rst:471]:
```suggestion
   a. On the leader kit, enable the SRP Server function:
```

Comment (edmont) [reply samples/openthread/cli/README.rst:471]: Thanks, C&P :)

File: samples/openthread/cli/README.rst [hunk 2]
```diff
@@ -454,14 +459,65 @@ To test the Thread 1.2 features, complete the following steps:
...
+         uart:~$ ot srp client host name my-host
+         Done
+         uart:~$ ot srp client host address fdde:ad00:beef:0:e0fc:dc28:1d12:8c2
+         Done
+         uart:~$ ot srp client service add my-service _ipps._tcp 12345
+         Done
+         uart:~$ ot srp client autostart enable
+         Done
+
+   #. Check if the host and service has been successfully registered:
```
Comment (wiba-nordic) [inline samples/openthread/cli/README.rst:491]:
```suggestion
   #. On the router kit (now SED), check that the host and service have been successfully registered:
```
Adding mention of the router kit is not 100 % required, but better to be extra thorough.

**Review by wiba-nordic — CHANGES_REQUESTED** [review]
Very small change needed.

**Review by wiba-nordic — APPROVED** [review]

**Review by lmaciejonczyk — APPROVED** [review]

**Pull request merged**